### PR TITLE
feat(aot) Add intersection observer

### PR DESCRIPTION
**Pull request by horymury**

### CODE REVIEW

File: alwaysontop/constants.js [hunk 1]
```diff
@@ -21,7 +21,9 @@ module.exports = {
         HIDE: 'aot-hide',
         OPEN: 'aot-open',
         SHOW: 'aot-show',
```
Comment (csett86) [inline alwaysontop/constants.js:23]: Quick (any maybe stupid) question: Whats the difference between the states "SHOW"/"HIDE" and "SHOW_AOT_WINDOW" / "HIDE_AOT_WINDOW"?

Comment (horymury) [reply alwaysontop/constants.js:23]: > Quick (any maybe stupid) question: Whats the difference between the states "SHOW"/"HIDE" and "SHOW_AOT_WINDOW" / "HIDE_AOT_WINDOW"?

SHOW/HIDE are events sent from main process to notify renderer process that AOT was shown/hidden, while "SHOW_AOT_WINDOW" / "HIDE_AOT_WINDOW" are events sent by the render process to tell the main process to show/hide the AOT. I wanted to keep the existing logic for show/hide AOT.

**Review by csett86 — APPROVED** [review]

File: alwaysontop/constants.js (hunk 1 above)
Comment (csett86) [reply alwaysontop/constants.js:23]: Thanks for the explanation

**Comment by csett86** [comment]
@horymury, to get a release out successfully incl npm, please see https://github.com/jitsi/jitsi-meet-electron-sdk#publishing

**Comment by horymury** [comment]
> @horymury, to get a release out successfully incl npm, please see https://github.com/jitsi/jitsi-meet-electron-sdk#publishing

Thank you, will do!

**Pull request merged**